src/main/webapp/app/entities/file-upload-exercise/file-upload-exercise-delete-dialog.component.html

```diff
@@ -8,11 +8,17 @@ <h4 class="modal-title" jhiTranslate="entity.delete.title">Confirm delete operat
         <p jhiTranslate="artemisApp.fileUploadExercise.delete.question" [translateValues]="{ title: fileUploadExercise.title }">
             Are you sure you want to delete this File Upload Exercise?
         </p>
+        <div class="form-group">
+            <label class="text-danger">Please type in the name of the Exercise to confirm.</label>
```

Comment:
The translation for this text is missing

Reply:
Yes, I added it now